nl_server/loader.py

```diff
@@ -90,15 +74,25 @@ def _load_yaml(flask_env: str) -> Dict[str, str]:
 
     # For custom DC dev env, only keep the default index.
     if _is_custom_dc_dev(flask_env):
+      default_model_name = embeddings_map['indexes'][
+          config.DEFAULT_INDEX_TYPE]['model']
       embeddings_map = {
-          config.DEFAULT_INDEX_TYPE: embeddings_map[config.DEFAULT_INDEX_TYPE]
+          'version': embeddings_map['version'],
+          'indexes': {
+              config.DEFAULT_INDEX_TYPE:
+                  embeddings_map['indexes'][config.DEFAULT_INDEX_TYPE]
+          },
+          'models': {
+              default_model_name: embeddings_map['models'][default_model_name]
+          }
       }
```

Comment:
Looks okay.  But I'm not sure why this exists, and may come in the way of UN testing etc, it seems @keyurva added this?